AnkiDroid/build.gradle

```
@@ -281,6 +281,11 @@ tasks.register('copyTestLibIntoAndroidTest', Copy) {
 tasks.named('preBuild').configure { dependsOn('copyTestLibIntoAndroidTest') }
 tasks.named('runKtlintCheckOverAndroidTestSourceSet').configure { mustRunAfter('copyTestLibIntoAndroidTest') }
 
+// to run manually: `./gradlew :testlib:assemblePlayDebugAndroidTest`
+tasks.named("assemblePlayDebugAndroidTest") {
```

The other task dependencies use `.configure` for a reason: it is done that way in order to work well with Gradle's ability to defer configuration, which can speed compilation for jobs that don't use some tasks and thus don't need to configure them

This was suggested as something to change via gradle lint rules in one of the recent gradle updates

https://github.com/ankidroid/Anki-Android/pull/14855/commits/68b1f68a9dd3950bd2a633af89175daea915a6e3

```suggestion
tasks.named("assemblePlayDebugAndroidTest").configure {
```